public/docs/ts/latest/glossary.jade

```diff
@@ -28,26 +28,25 @@ block includes
 +ifDocsFor('ts')
   a#aot
   :marked
-    ## Ahead of Time (AOT) Compilation
+    ## Ahead of Time (AOT) compilation
```

Comment:
In fact, let's make this "Ahead-of-time", since it's a adjectival phrase before a noun. (http://www.grammarbook.com/punctuation/hyphens.asp)